### ISSUE TITLE
Handle h5 files of length different from 3600s

### ISSUE THREAD

**Issue by mikailweston** [jcornford/pyecog#35]

**Comment by mikailweston** [comment]
Very important

**Comment by raubsau** [comment]
I would like to be able to change the length (or maybe even autodetect) the length of NDF files.

**Comment by jcornford** [comment]
yes, I think we should have options to force length or autodetect it. For classification, I think there will be hardcoded places that expect 3600/chunk_len, so there might be a few problems intially
